hoomd/mpcd/RadialSolventVelocityAnalyzer.cc

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2009-2019 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file mpcd/RadialSolventVelocityAnalyzer.cc
+ * \brief Declaration of mpcd::RadialSolventVelocityAnalyzer
+ */
+
+#include "RadialSolventVelocityAnalyzer.h"
+#include "hoomd/extern/pybind/include/pybind11/stl.h"
+
+namespace mpcd
+{
+
+/*!
+ * \param sysdata MPCD system data
+ * \param bin_width Width for binning particle distances
+ */
+RadialSolventVelocityAnalyzer::RadialSolventVelocityAnalyzer(std::shared_ptr<mpcd::SystemData> sysdata,
+                                                             Scalar bin_width)
+    : m_mpcd_sys(sysdata),
+      Analyzer(m_mpcd_sys->getSystemDefinition()),
+      m_mpcd_pdata(m_mpcd_sys->getParticleData()),
+      m_sysdef(m_mpcd_sys->getSystemDefinition()),
+      m_global_box(m_sysdef->getParticleData()->getGlobalBox()),
+      m_bin_width(bin_width),
+      m_Rmax(0.0),
```

Comment:
Don't set the bin width here, as it may actually change once you use `R` and `bin_width` to compute the number of bins. (I see this is a small error in `RDFAnalyzer` too, if you chose `R` not to be a multiple of `bin_width`.)
```suggestion
      m_R(R),
```